### PR TITLE
gimp-help: update to  2.10.34 and added the homepage

### PR DESCRIPTION
**Pull request by FAb7D**
**Summary**  
- The inclusion of "homepage" to package.yml
- Part of https://github.com/getsolus/packages/issues/411
- Update to new  source version
- Added quickreferences in pdf format
- Added new languages
- Limit the jobs to 8 so as to not overwhelm the build server

**Test Plan**

- Launched with Gimp, checked the folder /usr/share/gimp/*/help/pdf/

**Checklist**

- [x] Packages was built and tested against unstable

### CODE REVIEW

**Comment by silkeh** [comment]
Your commit and PR are missing a title and are strangely indented. Can you update both?

**Comment by FAb7D** [comment]
Is it ok now?

File: packages/g/gimp-help/package.yml [hunk 1]
```diff
@@ -1,72 +1,163 @@
...
+          - /usr/share/gimp/*/help/pdf/gimp-keys-zh_CN.pdf
 builddeps  :
     - pkgconfig(gimp-2.0)
     - docbook-xml
+    #The following have been added to populate the Quickreferences
+    - pkgconfig(librsvg-2.0)
+    - font-noto-cjk
 setup      : |
     %configure
 build      : |
```
Comment (ReillyBrogan) [inline packages/g/gimp-help/package.yml:158]: I built this locally on my system that has 16 cores (32 threads) and observed that the system ran out of memory and OOMkilled the build. I lowered it to 16GB and observed a memory usage of ~80GB, higher than the build server has. Please replace the `build` section with the following:
```
    # The memory demands for this build are very high. -j16 was observed consuming ~80GB of memory, more than the build server has.
    # Limit the jobs to 8 so as to not overwhelm the build server.
    MAX_JOBS=%YJOBS%
    if [ $MAX_JOBS -gt 8 ]; then
        MAX_JOBS=8
    fi
    make -j$MAX_JOBS
```

File: packages/g/gimp-help/package.yml [hunk 2]
```diff
@@ -1,72 +1,163 @@
 name       : gimp-help
```
Comment (ReillyBrogan) [inline packages/g/gimp-help/package.yml:1]: The commit title and PR title needs to have lowercase `gimp-help`, not `Gimp-help`.

**Review by ReillyBrogan — CHANGES_REQUESTED** [review]

File: packages/g/gimp-help/package.yml (hunk 2 above)
Comment (FAb7D) [reply packages/g/gimp-help/package.yml:1]: Done, I think it's fixed now

File: packages/g/gimp-help/package.yml (hunk 1 above)
Comment (FAb7D) [reply packages/g/gimp-help/package.yml:158]: Yes, in fact I had to change eopkg.conf from jobs=auto to jobs=3...

Comment (FAb7D) [reply packages/g/gimp-help/package.yml:158]: Done

File: packages/g/gimp-help/package.yml (hunk 2 above)
Comment (FAb7D) [reply packages/g/gimp-help/package.yml:1]: ?

**Pull request closed without merging**